src/Land/Model/soil_water.jl

```diff
@@ -89,7 +90,7 @@ end
         moisture_factor::AbstractMoistureFactor{FT} = MoistureIndependent{FT}(),
         hydraulics::AbstractHydraulicsModel{FT} = vanGenuchten{FT}(),
         initialϑ_l = (aux) -> FT(NaN),
-        initialθ_ice = (aux) -> FT(NaN),
+        initialθ_i = (aux) -> FT(NaN),
```

Comment:
could we change this to match line 107? (i.e. not NaN, but 0.0)

Reply:
done :)